conferences.json

```diff
@@ -224,4 +224,20 @@
             "price": "99 USD",
         }
     },
+    {
+        "name": "Cloud Native Rust Day",
+        "date": "2021-05-03T00:00",
```

Comment:
This needs a timezone.

Reply:
Specifically looked it up bot forgot to add it 🤦